src/guides/v2.4/release-notes/release-notes-2-4-0-open-source.md

```diff
@@ -1746,123 +1795,119 @@ We have fixed hundreds of issues in the Magento 2.4.0 core code.
+*  The submenu in the Admin Marketing tab now automatically adjusts to the length of menu items as expected.  You can also now scroll horizontally.  _Fix submitted by Paweł Tylek in pull request [26034](https://github.com/magento/magento2/pull/26034)_. [GitHub-18687](https://github.com/magento/magento2/issues/18687)
 
-<!--- ENGCOM-6376-->
+<!--- ENGCOM-6519-->
 
-*  Scrolling of product  pages now works as expected in mobile view.  _Fix submitted by Mateusz Krzeszowiak in pull request [25385](https://github.com/magento/magento2/pull/25385)_. [GitHub-10518](https://github.com/magento/magento2/issues/10518), [GitHub-21717](https://github.com/magento/magento2/issues/21717)
+*  The table `quote`  column `customer_note` now uses type `text` as expected. Previously, it used type `varchar(255)`. _Fix submitted by Ravi Chandra in pull request [26160](https://github.com/magento/magento2/pull/26160)_. [GitHub-26155](https://github.com/magento/magento2/issues/26155)
 
-<!--- ENGCOM-6622-->
+<!--- ENGCOM-7153-->
 
-*  The Compare Products section of the Catalog page is now displayed as expected in mobile view on displays with resolutions of less than 767px. _Fix submitted by Hitesh in pull request [26418](https://github.com/magento/magento2/pull/26418)_. [GitHub-26416](https://github.com/magento/magento2/issues/26416)
+*  Corrected display issues with the table header on the **My Account**  > **Address Book**  Address Entries table.  _Fix submitted by Abrar Pathan in pull request [27336](https://github.com/magento/magento2/pull/27336)_. [GitHub-27335](https://github.com/magento/magento2/issues/27335)
 
-<!--- ENGCOM-7137-->
+<!--- ENGCOM-5986-->
 
-*  You can now update the value of a downloadable product’s sample and link title by enabling the  **Use default** checkbox. _Fix submitted by Abel Truong in pull request [27295](https://github.com/magento/magento2/pull/27295)_. [GitHub-27169](https://github.com/magento/magento2/issues/27169)
+*  The currency symbol used in Layered Navigation Price Step is now based on default settings as expected.  _Fix submitted by Bartłomiej Szubert in pull request [24815](https://github.com/magento/magento2/pull/24815)_. [GitHub-21684](https://github.com/magento/magento2/issues/21684)
 
-<!--- ENGCOM-6532-->
+<!--- ENGCOM-6324-->
 
-*  Corrected spacing between the **Update** button and **Qty** field on the mini cart.  _Fix submitted by Hitesh in pull request [26234](https://github.com/magento/magento2/pull/26234)_. [GitHub-26227](https://github.com/magento/magento2/issues/26227)
+*  The symbol for the Belarusian currency (BYR) has been updated. _Fix submitted by Bartomiej Szubert in pull request [25723](https://github.com/magento/magento2/pull/25723)_. [GitHub-24713](https://github.com/magento/magento2/issues/24713)
```

Comment:
In the previous item, Bartomiej contains a non-Latin character

Reply:
Corrected throughout both versions of the notes